### DISCUSSION TITLE
DAX STUDIO:  an error occurred reading the system locales

### DISCUSSION THREAD

**Discussion by alceufec** [DaxStudio/DaxStudio#1015] [Q&A]
Hi, I'm new to DAX and I just installed in my PC, W11, DAX Studio 3.0.5. But before I connect with any power bi file a get the message in the log window: "An error occurred reading the system locales: Já existe uma entrada com a mesma chave."Já existe uma entrada com a mesma chave" means something like: there is already one entry with the same key. What does it mean? What should I do to fix this? 
In fact, if a try to connect to any PBI file the app crashes and close. I would appreciate any help! Thanks.

**Comment by dgosbell** [comment]
This is probably 2 separate issues, the system locales message is a warning that I could not populate the locale drop down as there appear to be multiple locales with the same ID on your system. I cannot reproduce this issue probably fix this by ignoring duplicates.

The crash is probably a separate issue. After the app crashes can you check the log folder ( see https://daxstudio.org/docs/troubleshooting/logging-settings/#log-folder-location) and upload the log file with the most recent date to this issue on Github?

**Reply by alceufec** [reply]
You are right when said there were 2 issues, the first one (error message before connecting do files) remains, but for any reason DAX studio stop to crash. Probably there was some windows update fixing some bug. Thank you.
